src/utils/cacheHandlers.ts

```diff
@@ -155,6 +157,14 @@ const stringToHash = (message: string): number => {
   return hash;
 };
 
+export const deleteExpiredCachedItemsAtInterval = (): void => {
+  // delete expired items on initialization
+  findAndDeleteExpiredCachedItems();
+  setInterval(() => {
+    findAndDeleteExpiredCachedItems();
+  }, CLEAR_CACHE_INTERVAL);
+};
+
 export const findAndDeleteExpiredCachedItems = async (): Promise<void> => {
```

Comment:
Nitpicking: This doesn't need to be exported since it's now used in only this file.

FFTR

Reply:
Removed the export.